frontend/app/ui/subscriptions/confirm/controller.js

```diff
@@ -17,12 +18,28 @@ export default class SubscriptionsConfirmController extends Controller {
 
   @action async accept(order) {
     const project = order.project.get("name");
+    const customer = order.project.get("customer.name");
+
+    try {
+      await UIkit.modal.confirm(
+        this.intl.t("page.subscriptions.confirm.prompt.accept", {
+          hours: order.duration.asHours(),
+          customer,
+          project,
+        })
+      );
+    } catch (error) {
+      return;
```

Comment:
why is this in a try catch if you then dont handle the error / give an info to the user?

Reply:
The "error" is when the user cancels the confirmation. I agree that this is rather weird. I would have expected a boolean as with [window.confirm](https://developer.mozilla.org/en-US/docs/Web/API/Window/confirm). But this is [how UIkit implemented](https://getuikit.com/docs/modal#modal-dialogs) this.

So. There is no real error to handle.

Reply:
Ah okay :ok_hand:

Reply:
I guess the reasoning is that you have to have a catch anyway.